static/panes/diff.js

```diff
@@ -33,22 +33,50 @@ var ga = require('../analytics');
 require('../modes/asm-mode');
 require('selectize');
 
-function State(id, model) {
+var
+    DiffType_ASM = 0,
+    DiffType_CompilerStdOut = 1,
```

Comment:
Wow super cool bonus feature of the compiler stdout and stderr!

Maybe worth noting in a comment here that these values are used in the state (and so need to be stable, else old URLs might break).